riot.d.ts

```diff
@@ -13,14 +13,24 @@ import {
 } from '@riotjs/dom-bindings'
 
 // Internal Types and shortcuts
-export type RegisteredComponentsMap = Map<string, () => RiotComponent>
+export type SlotBinding = {
```

Comment:
This should be moved to the dom-bindings repo https://github.com/riot/dom-bindings/blob/3b48c30e6bc89f9f71ca84fb8d3c06fa0cced357/dom-bindings.d.ts#L98-L102

Let's avoid duplication and consume this type in Riot.js importing it from the dom-bindings